### PR TITLE
chore: Redesign onboarding screen

### PR DESCRIPTION
**Pull request by fayazara**
A revamp of the onboarding screen
CleanShot 2024-08-12 at 19 51 22@2x
CleanShot 2024-08-12 at 19 51 11@2x

### CODE REVIEW

File: app/javascript/dashboard/components/widgets/conversation/OnboardingView.vue [hunk 1]
```diff
@@ -25,81 +38,115 @@ export default {
...
+        <p
+          class="text-base text-slate-800 dark:text-slate-100 font-interDisplay font-semibold tracking-[0.3px]"
+        >
+          {{ $t('ONBOARDING.ALL_CONVERSATION.TITLE') }}
+        </p>
+        <p class="text-slate-600 dark:text-slate-400 text-sm">
+          {{ $t('ONBOARDING.ALL_CONVERSATION.DESCRIPTION') }}
+        </p>
+      </div>
+    </div>
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/widgets/conversation/OnboardingView.vue:76]: Create a component out of this. The code is repeated.

File: app/javascript/dashboard/components/widgets/conversation/OnboardingView.vue [hunk 2]
```diff
@@ -10,7 +10,20 @@ export default {
...
+    greeting() {
+      const time = new Date();
+      const hours = time.getHours();
+      if (hours < 12) return 'Good morning';
+      if (hours < 18) return 'Good afternoon';
+      return 'Good evening';
+    },
+    greetingMessage() {
+      return `👋 ${this.greeting}, ${this.currentUser.name}. Welcome to ${this.globalConfig.installationName}.`;
+    },
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/widgets/conversation/OnboardingView.vue:26]: Use i18n. Read the guidelines for translation strings. Do not break the strings. https://chatwoot.help/hc/handbook/articles/1710791123-translation-guidelines

File: app/javascript/dashboard/components/layout/sidebarComponents/AccountSelector.vue [hunk 3]
```diff
@@ -69,10 +69,7 @@ export default {
       </div>
     </div>
 
-    <div
-      v-if="globalConfig.createNewAccountFromDashboard"
-      class="flex items-center justify-end gap-2 px-8 pt-4 pb-8"
-    >
+    <div class="flex items-center justify-end gap-2 px-8 pt-4 pb-8">
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/layout/sidebarComponents/AccountSelector.vue:72]: Please bring back the config. The button is for internal testing only, we have not given this option to anyone.

**Review by pranavrajs — CHANGES_REQUESTED** [review]
Please take a look at the comments.

**Comment by pranavrajs** [comment]
On the design side, the first card not having a link seems a bit off. Align the text to the top, leaving space below if you can't find a good link to add.

Screenshot 2024-08-12 at 9 22 36 PM

File: app/javascript/dashboard/components/layout/sidebarComponents/AccountSelector.vue (hunk 3 above)
Comment (fayazara) [reply app/javascript/dashboard/components/layout/sidebarComponents/AccountSelector.vue:72]: This got slipped in accidentally, will add it back

File: app/javascript/dashboard/components/widgets/conversation/OnboardingView.vue [hunk 4]
```diff
@@ -1,107 +1,103 @@
...
+      if (hours < 12) return this.$t('ONBOARDING.MORNING');
+      if (hours < 18) return this.$t('ONBOARDING.AFTERNOON');
+      return this.$t('ONBOARDING.EVENING');
+    },
+    greetingMessage() {
+      return this.$t('ONBOARDING.GREETING', {
+        greeting: this.greeting,
+        name: this.currentUser.name,
+        installationName: this.globalConfig.installationName,
+      });
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/widgets/conversation/OnboardingView.vue:33]: This is still problematic. The sentence is broken. You should not pass anything other than numbers, names as variables. Here `Good morning / evening` is passed as a variable to the sentence. Please read this guideline properly. 
https://www.chatwoot.com/hc/handbook/articles/1710791123-translation-guidelines

File: public/dashboard/images/onboarding/inboxes.png
Comment (pranavrajs) [inline public/dashboard/images/onboarding/inboxes.png:1]: Unused file

File: public/dashboard/images/onboarding/omnichannel.png
Comment (pranavrajs) [inline public/dashboard/images/onboarding/omnichannel.png:1]: unused file.

File: app/javascript/dashboard/components/widgets/conversation/OnboardingView.vue [hunk 5]
```diff
@@ -113,6 +109,7 @@ export default {
   overflow: auto;
   text-align: left;
 }
+
 .onboarding {
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/widgets/conversation/OnboardingView.vue:113]: Please remove unused CSS.

File: app/javascript/dashboard/components/widgets/conversation/OnboardingView.vue [hunk 6]
```diff
@@ -1,107 +1,103 @@
 <script>
 import { mapGetters } from 'vuex';
 import { useAccount } from 'dashboard/composables/useAccount';
+import OnboardingFeatureCard from './OnboardingFeatureCard.vue';
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/widgets/conversation/OnboardingView.vue:4]: Update this to component to use <script setup> if possible.

File: app/javascript/dashboard/components/widgets/conversation/OnboardingFeatureCard.vue [hunk 7]
```diff
@@ -0,0 +1,55 @@
+<script>
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/widgets/conversation/OnboardingFeatureCard.vue:1]: setup

File: app/javascript/dashboard/components/widgets/conversation/OnboardingView.vue (hunk 4 above)
Comment (fayazara) [reply app/javascript/dashboard/components/widgets/conversation/OnboardingView.vue]: @pranavrajs Updated, how does this look?

**Comment by fayazara** [comment]
> Please see the individual comments.

@pranavrajs Made the changes

**Review by pranavrajs — APPROVED** [review]

**Pull request merged**